fuzz/fuzz_targets/decompress_lzma.rs

```diff
@@ -0,0 +1,18 @@
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+extern crate lzma_rs;
+
+use lzma_rs::error::Result;
+
+fn decode_lzma(compressed: &[u8]) -> Result<Vec<u8>> {
+    let mut bf = std::io::Cursor::new(compressed);
```

Comment:
In tests, I've been using `std::io::BufReader` around a slice. Which one of a `Cursor` or `BufReader` is more appropriate? The seeking property of `Cursor` is not needed, but is the `BufRead` implementation more efficient for a `Cursor<&[u8]>` or for a `BufReader<&[u8]>` (or are they the same)?

Reply:
Swapping one for the other didn't result in any measurable difference in fuzzing speed.